EIPS/eip-7702.md

```diff
@@ -46,7 +46,7 @@ authorization_list = [[chain_id, address, nonce, y_parity, r, s], ...]
 
 Transaction is considered invalid if authorization list items can't be decoded as:
 
-* `chain_id`: unsigned 256-bit integer.
+* `chain_id`: Value 0 or 1.
```

Comment:
```suggestion
* `chain_id`: chain's current ID or zero
```

Reply:
cc @lightclient ?

Reply:
since the chain's ID can really be an uint256, I don't see the need to add this

Reply:
We should check for chain_id or zero here, not for 0 or 1 (because this would invalidate all txs on chain ids which are not 1, and which txs are specifically signed for that chain) - this would invalidate all specific devnet-txs signed with nonzero chain id.